tests/test_files/a_non_recursive.txt

```diff
@@ -0,0 +1 @@
+a_non_recursive
```

Comment:
do you need any of these files pre-existing, surely you just need to:
* create a new file in a sub directory and show it isn't detected with `recursion=False`
* create a new file in a sub directory and show it is detected with `recursion=True`

?

Reply:
I wrote tests following the standards for before

-  a_non_recursive.txt is for test_modify_write_non_recursive
- c_non_recursive.txt  is for test_delete_non_recursive

So that modify and delete were also included in the test. I included new files because the pytest fixture is set to session, therefore c.txt cannot be deleted by two different tests. If set to function, it goes through.

What do you think, should instead name them to d.txt and e.txt or use the same files but with `@pytest.fixture(scope='function')`?